code/src/components/ArtistName.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const ArtistName = (props) => {
+  const artistArray = props.artists;
+  const allArtists = artistArray.map((artist, index) => {
+    // If the artist is in the last place of the array and the array of artists is longer than 1 add
+    // '&' before artistname.
+    if (index === artistArray.length - 1 && artistArray.length !== 1) {
```

Comment:
Helpful comments to follow along the code. I would love to learn more behind this first part to generate the '&' symbol. Good job!